config/daemon/prometheus.md

```diff
@@ -36,7 +36,7 @@ If the file is currently empty, paste the following:
 
 ```json
 {
-  "metrics-addr" : "127.0.0.1:9323",
+  "metrics-addr" : "0.0.0.0:9323",
```

Comment:
I'll need to have a closer look, but this changes the metrics to be accessible on "any" IP address, not just the loopback IP address.

Depending on the setup, this may be what's wanted, but it also means metrics (which potentially could include sensitive data?) are exposed publicly.